src/quafel/pipelines/data_science/frameworks.py

```diff
@@ -349,3 +259,311 @@ def get_result(self) -> Dict[str, float]:
+                cir.add_Z_gate(mapping[int(ary[0])])
+            elif instr[0:2] == "hq":
+                matchobj = re.match(r"hq\[(\d+)\];", instr)
+                assert matchobj is not None
+                ary = matchobj.groups()
+                cir.add_H_gate(mapping[int(ary[0])])
+            elif instr[0:2] == "sq":
+                matchobj = re.match(r"sq\[(\d+)\];", instr)
+                assert matchobj is not None
+                ary = matchobj.groups()
+                cir.add_S_gate(mapping[int(ary[0])])
+            elif instr[0:4] == "sdgq":
+                matchobj = re.match(r"sdgq\[(\d+)\];", instr)
+                assert matchobj is not None
+                ary = matchobj.groups()
+                cir.add_Sdag_gate(mapping[int(ary[0])])
+            elif instr[0:2] == "tq":
+                matchobj = re.match(r"tq\[(\d+)\];", instr)
+                assert matchobj is not None
+                ary = matchobj.groups()
+                cir.add_T_gate(mapping[int(ary[0])])
+            elif instr[0:4] == "tdgq":
+                matchobj = re.match(r"tdgq\[(\d+)\];", instr)
+                assert matchobj is not None
+                ary = matchobj.groups()
+                cir.add_Tdag_gate(mapping[int(ary[0])])
+            elif instr[0:2] == "rx":
+                matchobj = re.match(
+                    rf"rx\(({GENERAL_NUMBER_PATTERN})\)q\[(\d+)\];", instr
+                )
+                assert matchobj is not None
+                ary = matchobj.groups()
+                cir.add_RX_gate(mapping[int(ary[1])], -float(ary[0]))
+            elif instr[0:2] == "ry":
+                matchobj = re.match(
+                    rf"ry\(({GENERAL_NUMBER_PATTERN})\)q\[(\d+)\];", instr
+                )
+                assert matchobj is not None
+                ary = matchobj.groups()
+                cir.add_RY_gate(mapping[int(ary[1])], -float(ary[0]))
+            elif instr[0:2] == "rz":
+                matchobj = re.match(
+                    rf"rz\(({GENERAL_NUMBER_PATTERN})\)q\[(\d+)\];", instr
+                )
+                assert matchobj is not None
+                ary = matchobj.groups()
+                cir.add_RZ_gate(mapping[int(ary[1])], -float(ary[0]))
+            elif instr[0:1] == "p":
+                matchobj = re.match(rf"p\({GENERAL_NUMBER_PATTERN}\)q\[(\d+)\];", instr)
+                assert matchobj is not None
+                ary = matchobj.groups()
+                cir.add_U1_gate(mapping[int(ary[1])], float(ary[0]))
+            elif instr[0:2] == "u1":
+                matchobj = re.match(
+                    rf"u1\(({GENERAL_NUMBER_PATTERN})\)q[(\d+)];", instr
+                )
+                assert matchobj is not None
+                ary = matchobj.groups()
+                cir.add_U1_gate(mapping[int(ary[1])], float(ary[0]))
+            elif instr[0:2] == "u2":
+                matchobj = re.match(
+                    rf"u2\(({GENERAL_NUMBER_PATTERN}),"
```

Comment:
The original code misses a '(' and ')' here. See https://github.com/qulacs/qulacs/blob/2750414e8e1ebc61064207158dd8b0e618708ad4/pysrc/qulacs/converter/qasm_converter.py#L120